qa/rpc-tests/checkdatasig-activation.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""
+This test checks activation of OP_CHECKDATASIG
+"""
+import test_framework.loginit
+from test_framework.test_framework import ComparisonTestFramework
+from test_framework.util import satoshi_round, assert_equal, assert_raises_rpc_error, start_node
+from test_framework.comptool import TestManager, TestInstance, RejectResult
+from test_framework.blocktools import *
+from test_framework.script import *
+
+# far into the future
+MAGNETIC_ANOMALY_START_TIME = 2000000000
```

Comment:
I's rather use another name for this variable. `Magnetic Anomaly`  is the name ABC chose to name the November upgrade, it is peculiar to ABC way of doing things this is not something specified in the spec.

Reply:
@ptschip is using `NOV152018_START_TIME` in #1376